src/containers/Extension/actions.js

```diff
@@ -1,3 +1,5 @@
 export { fetchNamespaces } from '../../actions/namespaces';
 export { fetchPipeline, fetchPipelines } from '../../actions/pipelines';
+export { fetchSecrets } from '../../actions/secrets';
 export { fetchTask, fetchTasks } from '../../actions/tasks';
+export { fetchServiceAccounts } from '../../actions/serviceAccounts';
```

Comment:
Can you move this up a line? Having the imports in alphabetical order like the rest makes it easier to scan